backend/faucet.go

```diff
@@ -140,8 +145,17 @@ func CheckAccountBalance(address string, amountFaucet string, key string) error
 		}
 	}
 
-	if (len(queryRes.Raw) != 0 && float64(queryRes.Raw[0].Balance) < float64(1000)*math.Pow(10, 6)) ||
-		accErr != nil {
+	if len(queryRes.Value.Coins) != 0 {
+		f, err := strconv.ParseFloat(queryRes.Value.Coins[0].Amount, 64)
+		if err != nil {
+			return nil
+		}
+		balance = f
+	} else {
+		return nil
+	}
+
+	if balance < float64(10)*math.Pow(10, 6) || accErr != nil {
```

Comment:
Use MAX_TOKENS_ALLOWED config variable. 
```suggestion
	if balance < MAX_TOKENS_ALLOWED || accErr != nil {
```